weed/pb/iam.proto

```diff
@@ -23,7 +23,7 @@ message Identity {
     string name = 1;
     repeated Credential credentials = 2;
     repeated string actions = 3;
-    Account account = 4;
+    string AccountId = 4;
 }
```

Comment:
In s3.configure, accessKey is associated with an Account through the AccountId, so only the AccountId field is needed